scripts/quests/jeuno/LB05_1_Shattering_Stars.lua

```diff
@@ -0,0 +1,147 @@
+-- Shattering Stars
+-----------------------------------
+-- Log ID: 3, Quest ID: 132
+-- Maat : !pos 8 3 118 243
+-----------------------------------
+require("scripts/settings/main")
+require("scripts/globals/items")
+require("scripts/globals/keyitems")
+require("scripts/globals/npc_util")
+require("scripts/globals/quests")
+require("scripts/globals/titles")
+require('scripts/globals/interaction/quest')
+local ID = require("scripts/zones/RuLude_Gardens/IDs")
+-----------------------------------
+local quest = Quest:new(xi.quest.log_id.JEUNO, xi.quest.id.jeuno.SHATTERING_STARS)
+-----------------------------------
+
+quest.reward =
+{
+    fame  = 80,
+    fameArea = JEUNO,
+    title = xi.title.STAR_BREAKER,
+}
+
+quest.sections =
+{
+    -- Section: Quest available.
+    {
+        check = function(player, status)
+            return status == QUEST_AVAILABLE and
+                player:getMainJob() <= 15 and -- Only the "old" jobs may start this quest.
+                player:getMainLvl() >= 66 and
+                player:getLevelCap() == 70 and
+                xi.settings.MAX_LEVEL >= 75
+        end,
+
+        [xi.zone.RULUDE_GARDENS] =
+        {
+            ['Maat'] =
+            {
+                onTrigger = function(player, npc)
+                    return quest:progressEvent(92, player:getMainJob())
+                end,
+            },
+
+            onEventFinish =
+            {
+                [92] = function(player, csid, option, npc)
+                    quest:begin(player)
+                end,
+            },
+        },
+    },
+
+    -- Section: Quest accepted.
+    {
+        check = function(player, status)
+            return status == QUEST_ACCEPTED and
+                player:getMainJob() <= 15 and
+                player:getMainLvl() >= 66
+        end,
+
+        [xi.zone.RULUDE_GARDENS] =
+        {
+            ['Maat'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') >= 1 then
+                        return quest:progressEvent(93) -- Complete quest.
+                    elseif quest:getVar(player, 'Prog') == 0 then
+                        return quest:event(91, player:getMainJob())
+                    end
+                end,
+                onTrade = function(player, npc, trade)
+                    local properTestimony = xi.items.WARRIORS_TESTIMONY + player:getMainJob() - 1
+                    if npcUtil.tradeHasExactly(trade, properTestimony) and quest:getVar(player, 'Prog') == 0 then
+                        return quest:progressEvent(64, player:getMainJob())
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [64] = function(player, csid, option, npc)
+                    if option == 1 then
+                        local mJob = player:getMainJob()
+                        if mJob == xi.job.MNK or mJob == xi.job.WHM or mJob == xi.job.SMN then
```

Comment:
This is a bit of a beefy conditional.  Not sure if there's a more simple way to handle this